src/Test/Integration/Model/Payment/FactoryTest.php

```diff
@@ -75,9 +80,12 @@ public function testPaymentMethodMapping($methodCode, $expectedPaymentMethod): v
     public function paymentMethodProvider(): array
     {
         return [
-            'return shopgate as default payment method' => ['PREPAY', 'shopgate'],
+            'return shopgate as default payment method' => ['SOMETHING', 'shopgate'],
+            'Bank Transfer'                             => ['PREPAY', 'banktransfer'],
+            'Cash on delivery'                          => ['COD', 'cashondelivery'],
+            'Invoice'                                   => ['INVOICE', 'checkmo'],
             'Braintree Credit Card'                     => ['BRAINTR_CC', 'braintree'],
-            'Braintree PayPal'                          => ['BRAINTR_PP', 'braintree_paypal']
+            'Braintree PayPal'                          => ['BRAINTR_PP', 'braintree_paypal'],
```

Comment:
this comma just keeps coming back ^_^ is it part of ur settings or is it the same bug I reported in bananas?